unity-renderer/Assets/DCLServices/PlacesAPIService/PlacesAPIService.cs

```diff
@@ -224,7 +224,7 @@ public async UniTask SetPlaceFavorite(Vector2Int coords, bool isFavorite, Cancel
 
         public async UniTask<bool> IsFavoritePlace(IHotScenesController.PlaceInfo placeInfo, CancellationToken ct, bool renewCache = false)
         {
-            var favorites = await GetFavorites(ct, renewCache);
+            var favorites = await GetFavorites(0,1000, ct, renewCache);
```

Comment:
I made it so that by passing -1, -1 it requests all favorites, otherwise I had to change the placesclient as well and needed to duplicate some code, let me know what you think

Reply:
Other option could be to make both int values as nullable and manage hte `null` values instead of `-1`, but it would be the same. Thanks for the change.